src/main/java/network/brightspots/rcv/CommonDataFormatReader.java

```diff
@@ -187,7 +186,7 @@ void parseXml(List<CastVoteRecord> castVoteRecords)
       if (contest.CVRContestSelection != null) {
         for (CVRContestSelection cvrContestSelection : contest.CVRContestSelection) {
           if (cvrContestSelection.Status != null && cvrContestSelection.Status
-              .equals(STATUS_NEEDS_ADJUDICATION)) {
+              .equals("needs-adjudication")) {
```

Comment:
what warning is this addressing? you can't define a constant unless it's used in multiple places?

Reply:
Yeah, the warning was recommending inlining a value that's only referenced once, which is consistent with Google's best practices as well (since it improves readability).